api/caching/tasks.py

```diff
@@ -131,11 +132,67 @@ def update_storage_usage_cache(target_id, target_guid, per_page=500000):
             offset += count
 
     key = cache_settings.STORAGE_USAGE_KEY.format(target_id=target_guid)
-    storage_usage_cache.set(key, storage_usage_total, cache_settings.FIVE_MIN_TIMEOUT)
+    storage_usage_cache.set(key, storage_usage_total, cache_settings.ONE_DAY_TIMEOUT)
```

Comment:
I agree with this. I put it in the defaults, had it reference cache settings, and had everything reference that setting rather than api.caching.settings. 👍